DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.CourseDelegates
+@using Microsoft.Extensions.Configuration
+@model CourseDelegatesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/courseDelegates.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Course delegates";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/TrackingSystem/Delegates/Shared/_DelegatesSideNavMenu.cshtml" model="DelegatePage.CourseDelegates" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Course delegates</h1>
+      </div>
+      <div class="nhsuk-grid-column-one-third heading-button-group">
+        <a class="nhsuk-button nhsuk-button--secondary heading-button" role="button">
+          Export
+        </a>
+      </div>
+    </div>
+
+    @if (Model.Courses.Any()) {
+      <div class="nhsuk-grid-row">
+        <form method="get" role="search">
+          <div class="nhsuk-grid-column-full nhsuk-form-group">
+            <label class="nhsuk-label--s nhsuk-u-margin-bottom-1">Choose course</label>
+            <select class="nhsuk-select nhsuk-search__input nhsuk-search__input--withdropdown course-dropdown"
+                    asp-for="CustomisationId"
+                    asp-items="Model.Courses"
+                    aria-label="Choose course"></select>
+            <button class="nhsuk-search__submit sort-submit" type="submit">
+              Select
+            </button>
+          </div>
+        </form>
+      </div>
+
+      @if (!Model.Active == true) {
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-full">
+            <p class="nhsuk-body-m">The currently selected course is inactive.</p>

```

Comment:
Could we use the NHS inset text component for this? https://service-manual.nhs.uk/design-system/components/inset-text